spec/decanter/parser/float_parser_spec.rb

```diff
@@ -8,8 +8,16 @@
 
   describe '#parse' do
     context 'with a string' do
-      it 'returns a float' do
-        expect(parser.parse(name, '1.00')).to match({name => 1.00})
+      context 'with a positive value' do
+        it 'returns a positive float' do
+          expect(parser.parse(name, '1.00')).to match({name => 1.00})
```

Comment:
While we're here, can we update the hashrocket syntax?

Reply:
interestingly, switching to symbol syntax breaks the tests

Reply:
Can you use `eq`? It looks like [match](https://relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/match-matcher) is meant for strings and regex. Is it possible that Rspec is coercing both into a string?

Reply:
Ohhhhh name is a variable. So you have to use hashrocket when the key is dynamic right? I think the match document I provided is a red herring, because it's actually `{ :foo => 1.00 }`

Reply:
nvm user error. this has to be `{name => 1.00}` because `name` is a variable, not the actual key

Reply:
ah, you beat me to it! I'm ok with being explicit